samples/gradle-multipart-sample/gradle-war-demo/build.gradle

```diff
@@ -12,8 +12,8 @@ repositories {
 }
 
 dependencies {
-    testImplementation group: 'junit', name: 'junit', version: '4.12'
-    providedCompile 'javax.servlet:javax.servlet-api:3.1.0'
-    compileOnly 'com.ibm.cics:com.ibm.cics.server:1.700.0-5.5-PH10453'
+    testImplementation group: 'junit', name: 'junit', version: '4.13.2'
+    providedCompile 'javax.servlet:javax.servlet-api:4.0.1'
```

Comment:
I think we likely don't want to be moving to a new major version of the servlet API